docs/usage/configuration/config_documentation.md

```diff
@@ -4033,9 +4042,9 @@ A worker can handle HTTP requests. To do so, a `worker_listeners` option
 must be declared, in the same way as the [`listeners` option](#listeners)
```

Comment:
tangential but I still don't like how the main proc has its own `listeners` option and other workers have their own special section.

Reply:
Tangential, to be sure. I'll put that down to work on later. I visualize that this means the `listeners` block would need a way to identify which listener applies to which worker, which means a (at the very least) restructure of how it's laid out. I'll try and workup some samples of how it could look to decide around.